Mathlib/Topology/Sets/Closeds.lean

```diff
@@ -331,6 +331,9 @@ lemma coe_finset_sup (s : Finset ι) (U : ι → Clopens α) :
   | empty => simp
   | insert _ IH => simp [IH]
 
+lemma coe_disjoint {s t : Clopens α} : Disjoint (s : Set α) t ↔ Disjoint s t := by
```

Comment:
```suggestion
@[simp, norm_cast]
lemma coe_disjoint {s t : Clopens α} : Disjoint (s : Set α) t ↔ Disjoint s t := by
```

Reply:
This seems to be completely unrelated to `Clopens`, it is true for every type carrying a `SetLike` structure. This is also true for some of the results above, but I'd advocate for a better home.

Reply:
I'm afraid this doesn't work. E.g. subgroups of a group a SetLike; but `Disjoint` in that case doesn't mean "intersection is the empty set" (because the empty set is not a subgroup), it means "intersection is {1}". There are very few set-like types in mathlib such that which the union, intersection, etc operations are fully compatible with coercion to sets.

Reply:
That's a very good point, yet I guess you agree that this has nothing to do with `Clopens`, no? I am just afraid that we keep repeating this everywhere for every kind of e.g. "special subspaces" in topological spaces.

Reply:
Oh, sure, very well: I was unaware of this Zulip discussion, and I certainly agree with you that once the community rejects one idea, we move on.